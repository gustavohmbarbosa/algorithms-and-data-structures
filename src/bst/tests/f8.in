8
66
1
50
1
20
1
10
1
30
1
40
1
15
1
31
8
15
8
31
8
50
8
30
8
60
99