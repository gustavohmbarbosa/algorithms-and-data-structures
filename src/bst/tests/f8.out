-1
10
40
-1
20
-1
